packages/syft/src/syft/ast/klass.py

```diff
@@ -161,6 +172,93 @@ def run_class_method(
 
         return result
 
+    def run_class_smpc_method(
+        __self: Any,
+        *args: Tuple[Any, ...],
+        **kwargs: Any,
+    ) -> object:
+        """Run remote class method on a SharePointer and get pointer to returned object.
+
+        Args:
+            *args: Args list of class method.
+            **kwargs: Keyword args of class method.
+
+        Returns:
+            Pointer to object returned by class method.
+        """
+        # relative
+        from ..core.node.common.action.smpc_action_message import (
```

Comment:
MAYBE : We could have the import at top of module.

Reply:
I get a recursive import :(